tour.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Tour</title>
+</head>
+<body>
+<div class="header">
+	<div class="wrapper">
+		<div class="top-header">
+			<div class="navigation">
+				<ul class="navigation-buttons">
+					<li><a href="#">About</a></li>
+					<li><a href="#">Audio</a></li>
+					<li><a href="#">Video</a></li>
+					<li><a href="#">Gallery</a></li>
+					<li><a href="#">Tour Dates</a></li>
+					<li><a href="#">Contacts</a></li>
+				</ul>
+			</div>
+		</div>
+		<div clas="header-left-content-block">
+			<div class="header-logo">
+				<a href="#" class="header-music-logo"><img src="images/logo.jpg" alt="music logo"></a>
+			</div>
+		</div>
+		<div clas="header-slider-wrapper">
+			<div class="header-slider">
+				<h1>Lilly Watson</h1>
+				<div class="header-slider-screen-resolution">
```

Comment:
please correct structure of slider and it's classnames:
http://image.prntscr.com/image/02a97155bae7461c8d59f9fbdd7fdb95.png
At the top you have a slider heading, left-right are slide arrows, center block is the content of current slide (or simply slide)